api/tests/Api/SnapshotTests/__snapshots__/ResponseSnapshotTest__testGetCollectionMatchesStructure with data set content_nodedefault_layouts__1.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
do we only have one DefaultLayout in the fixtures?

Reply:
No, one camp unrelated and one connected to an actual camp